content/docs/reference-javascript-environment-requirements.md

```diff
@@ -1,14 +1,14 @@
 ---
 id: javascript-environment-requirements
-title: JavaScript Environment Requirements
+title: जावास्क्रिप्ट पर्यावरण आवश्यकताएँ
```

Comment:
`पर्यावरण` changes the contextual meaning of environment. let keep it

`पर्यावरण` => `एनवायरनमेंट`